lib/requests_signature_dart.dart

```diff
@@ -16,3 +16,7 @@ export 'src/core/interface/signature_body_source_builder.dart';
 
 // Export default constants
 export 'src/core/default_constants.dart';
+
+// Export client implementation
+///Client: Dio
```

Comment:
Missing a space between the comment and `///`. Also at line 17, should use a triple slash instead?